crates/fonticulus/src/basictables.rs

```diff
@@ -164,6 +165,15 @@ pub fn compile_os2(
         .open_type_os2_subscript_x_size
         .unwrap_or((upm * 0.65).round() as i32) as i16;
 
+    let unicodes:HashSet<u32> = mapping.keys().cloned().collect();
```

Comment:
Thanks!

Which approach would you say is the best to adopt for the future? personally I find the turbofish style a little weird but for all I know this may be the most common.

Reply:
I personally probably do the second form more often, but I wouldn't comment on the first form in code review 🤷